### PR TITLE
FRONT-718: Implement chatroom view

### PR DESCRIPTION
**Pull request by mxinburritos**

### CODE REVIEW

File: src/components/Sidebar/Room.module.scss [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
-    background-color: rgb(255, 255, 255);
-    border: none;
-    height: 92px;
-    width: 350px;
-    border-radius: 20px;
+  margin: 8px;
+  text-align: left;
+  padding: 20px;
+  font-size: 18px;
+  font-family: 'Karelia';
```
Comment (fonty1) [inline src/components/Sidebar/Room.module.scss:6]: @mxinburritos should this have any fallback fonts or is it handled somewhere else?

**Review by fonty1 — APPROVED** [review]
LGTM 👍

**Comment by fonty1** [comment]
@mxinburritos I did merge 717 first, but looks like there's some conflicts on this one - can you fix and merge please?

**Comment by mxinburritos** [comment]
yep will take a look now!

**Pull request merged**